app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -0,0 +1,264 @@
+                    .apply();
+        } catch (IOException e) {
+            LOGGER.error("Couldn't get profiles from database at the onCreate", e);
+        }
+        LOGGER.info("SettingsActivity created");
+    }
+
+    @Override
+    public void onHeaderClick(final Header header, final int position) {
+        super.onHeaderClick(header, position);
+        if (header.id == R.id.key_management_header) {
+            final Intent intent = new Intent(this, KeyManagementActivity.class);
+            intent.putExtra(ConstantKeywords.SERVICE_LOCATOR, serviceLocator);
+            this.startActivity(intent);
+        }
+    }
+
+    /**
+     * Set up the {@link android.app.ActionBar}, if the API is available.
+     */
+    private void setupActionBar() {
+        final ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            // Show the Up button in the action bar.
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(final MenuItem item) {
+        final int id = item.getItemId();
+        if (id == android.R.id.home) {
+            finish();
+            return true;
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+    public void onBuildHeaders(final List<Header> target) {
+        loadHeadersFromResource(R.xml.pref_headers, target);
+    }
+
+    /**
+     * This method stops fragment injection in malicious applications.
+     * Make sure to deny any unknown fragments here.
+     *
+     * @param fragmentName The name of the fragment
+     * @return A boolean which is true when the fragment is valid
+     */
+    protected boolean isValidFragment(final String fragmentName) {
+        return PreferenceFragment.class.getName().equals(fragmentName)
+                || GeneralPreferenceFragment.class.getName().equals(fragmentName)
+                || ProfilePreferenceFragment.class.getName().equals(fragmentName);
+    }
+
+    /**
+     * This fragment shows general preferences only. It is used when the
+     * activity is showing a two-pane settings UI.
+     */
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+    public static final class GeneralPreferenceFragment extends PreferenceFragment {
+        @Override
+        public void onCreate(final Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            addPreferencesFromResource(R.xml.pref_general);
+            setHasOptionsMenu(true);
+
+            // Bind the summaries of EditText/List/Dialog/Ringtone preferences
+            // to their values. When their values change, their summaries are
+            // updated to reflect the new value, per the Android Design
+            // guidelines.
+            bindPreferenceSummaryToValue(findPreference("choose_verification_use_every_time"));
```

Comment:
Change "choose_verification_use_every_time" into "choose_verification_method_every_time"

Also, maybe use a enum for this?